src/background.js

```diff
@@ -7,3 +7,13 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
         chrome.action.setPopup({ tabId, popup: "popup_disabled.html" });
     };
 });
+
+chrome.runtime.onInstalled.addListener(function (object) {
```

Comment:
Can this be a page on this repository instead? Chrome and firefox both review URLs used in here, and I think they may not be happy with this one :)

Reply:
Google Docs is definitely better than notion! However, can you copy the contents into a markdown file in this repository (as part of this PR), and link to that instead?